script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt

```diff
@@ -11,47 +11,49 @@ Then―――what should I do?
 [sha:41014532b9f62efb0952798b567e8bbea2f50632]{
 -- Page 10034, Offset 34385.
 -- <痙攣|けいれん>する指で眼鏡の<蔓|つる>に触れる。
-I touch the frame of my glasses with my trembling fingers.
+My trembling fingers brush against the frame of my glasses.
 }
 [sha:96c8bf1983400d69685fa23c5cf4b198f952d305]{
 -- Page 387, Offset 34386.
 -- 「……できるわけないだろ、そんなマネ―――！」
-"...There's no way I can do something like that―――!"
+"No, there's no way I could do something like that―――!"
 }
 [sha:5858d322a9c9adbb0319dc5966efc9c93b4c0c90]{
 -- Page 387, Offset 34387.
 -- 激情に任せて手にしたモノを放り投げた。
-In a fit of rage, I end up throwing away the object in my hand.
+Overcome by emotion, I cast off the object in my hand.
 }
 [sha:f142cc2ebb96b1378181c6acc95a43a36116ea79]{
 -- Page 387, Offset 34388.
 -- 自分自身に殺意を覚えた結果だ。
-This is what my intention of killing myself has led to.
+I acted this way because I've hammered into myself that I have to die.
 }
 [sha:c7b09d94da4253ebdfc6998b51cbfc4dd96f079b]{
 -- Page 387, Offset 34389.
 -- カラン、カラン、と硬い物が暗い夜に転がっていく。
-The hard object rolls through the dead of night.
+The hard object clatters across the floor and disappears into the night.
 }
 [sha:f8bbdcc2df87681eecba7c9d3649a160a36c95fb]{
 -- Page 388, Offset 34390.
 -- ……線など見えない。
-...I don't see any lines.
+I don't see any lines...
 }
 [sha:2679c779f5918948fa0039ca6591547c88119f36]{
 -- Page 388, Offset 34391.
 -- 放り投げたものは、持っていたナイフだ。
-What I threw away was my knife.
+The object I hurled away from me was the knife that had been clutched in my hand.
 }
 [sha:ddda5b999aaca2f2e3db566ebba66b49acf48482]{
 -- Page 388, Offset 34392.
 -- 眼鏡は外さない。……絶対に、外さない。
-I won't take off my glasses... I will never take them off no matter what.
+I won't take off my glasses...
+ I won't take them off no matter what.
 }
 [sha:c203290801faa15aa9f33257046ca2200b24acb3]{
 -- Page 388, Offset 34393.
 -- ただ、俺はもう自信がないから、ナイフを遠くに投げただけだ。持っていればきっと、俺は自分が死ぬよりイヤな事をしてしまう。
-I threw my knife far away for the sole reason that I can't trust myself with it anymore. If I kept carrying it around, I'm sure I would've been met with a fate worse than death.
+I threw my knife far away for the sole reason that I can't trust myself with it anymore.
+ If I kept hold of it, I'm sure I would have done something far worse than simply dying.
```

Comment:
Hmm, reads kinda off to me like that. Like the tactile sensation of it would be what he dislikes, not the fact that it's something he would regret more.

Reply:
Yeah, it does read kinda off. I think I just didn't quite nail what I was going for.
>...done something that feels far worse than dying.

That said, I do think the personal aspect is a bit missing. He's the one who doesn't want to die, and he's the one who doesn't like doing the alternative here. **俺は自分が**死ぬよりイヤな事をしてしまう. See bolded. Very much about that shit.

Alternative
>If I kept hold of it, I'd have done something I could stand even less than my own death.